targets/python/Archive/PlayFabJson.py.ejs

```diff
@@ -0,0 +1,24 @@
+def serialize_instance(obj):
+    d = { '__classname__' : type(obj).__name__ }
+    d.update(dir(obj)) 
+    # o = vars(obj.__dict__)
```

Comment:
Can you clarify - are you intentionally leaving commented code in this review? Or should this be removed?

Reply:
The things in the "archive" folder are incomplete.  This is work that's half finished, and we're putting off to a future release.